test/parallel/test-dgram-socket-buffer-size.js

```diff
@@ -72,3 +72,22 @@ const dgram = require('dgram');
     socket.close();
   }));
 }
+
+function checkBufferSizeError(type, size) {
+  const errorObj = {
+    code: 'ERR_SOCKET_BUFFER_SIZE',
+    type: Error,
+    message: `Could not get or set buffer size: Error: EINVAL: invalid argument, uv_${type}_buffer_size` // eslint-disable-line max-len
```

Comment:
I think both of the long lines in this file could be handled without disabling the ESLint rule.